services/lumberjack/src/service.js

```diff
@@ -286,7 +291,8 @@ export default class Service {
     await this._influx.writeMeasurement('telemetry', [
       {
         fields: { pstatus },
-        tags: { host: this._telemetryHost, port: this._telemetryPort }
+        tags: { host: this._planeTelemetryHost, 
+          port: this._planeTelemetryPort }
```

Comment:
This is... [WET](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)

Reply:
Yeah, should probably make one function that handles plane/ground telemetry and have it let you pass in the host / port / measurement name so you can make two of them (one for the plane and ground each).